### PR TITLE
stream: improvements, example

### PR DESCRIPTION
**Pull request by atk**
Using audio context requires user interaction. If we instantiate the audio context before a user interaction, it will never run. Resuming it afterwards should help.

Also, this includes a test page now.

### CODE REVIEW

**Review by thetarnav — COMMENTED** [review]
Nice one. I haven't used that package, so I cannot comment on the logic improvemnts. But here are some other things :)

File: packages/stream/src/index.ts [hunk 1]
```diff
@@ -86,7 +86,10 @@ export const createStream = (
  * The stream will be stopped on cleanup automatically.
  */
 export const createAmplitudeStream = (
-  device: MediaDeviceInfo | Accessor<MediaDeviceInfo>
+  device?: 
+    | MediaDeviceInfo
+    | MediaStreamConstraints
+    | Accessor<MediaDeviceInfo | MediaStreamConstraints | undefined>
```
[the remark this answers is not in the document]
Comment (atk) [reply packages/stream/src/index.ts:92]: Good idea.

File: packages/stream/dev/vite.config.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "wrapper-v": "wrapper-h flex-col space-x-0 space-y-4",
+        node: "p-4 bg-orange-600 rounded m-2"
+      }
+    })
+  ],
+  build: {
+    target: "esnext",
+    polyfillDynamicImport: false
+  }
+});
```
[the remark this answers is not in the document]
Comment (atk) [reply packages/stream/dev/vite.config.ts:24]: done.

File: packages/stream/CHANGELOG.md [hunk 3]
```diff
@@ -13,3 +13,7 @@ Released a version with CJS and SSR.
 0.1.0
 
 Updated for latest Solid.
+
+0.1.1
+
+Fix for audio context requiring user interaction.
```
[the remark this answers is not in the document]
Comment (atk) [reply packages/stream/CHANGELOG.md]: OK, got it.

**Pull request merged**